.tool-versions

```diff
@@ -1,2 +1,2 @@
-elixir 1.5.2
-erlang 20.0
+elixir 1.7.3-otp-21
```

Comment:
the elixir version looks off - shouldn't it just be 1.7.3?

Reply:
You can now specify an Elixir Version that was compiled with a certain OTP version in this way. I find doing this makes Dialyzer much happier. It’s a pretty sweet asdf feature.

Reply:
Interesting, thanks.

Not just dialyzer but some other dependency compilation things I imagine...